tests/test_velocity_verlet_integrator.py

```diff
@@ -98,7 +98,7 @@ def test_reversibility():
     dt = 1.5e-3
 
     # Is not infinitely reversible, will fail after 3000 steps due to accumulation of coords/vels in floating point
-    for n_steps in [1, 10, 100, 500, 1000, 2000]:
+    for n_steps in [1, 10, 100, 500]:
```

Comment:
max n_steps = 100 or 500 seems ample to me

(main motivation of this test was to assert suitability for use in HMC, which (1) is not yet incorporated in TM, (2) would not need to use infinitely long proposal trajectories, (3) could use an on-the-fly reversibility check if absolutely needed, as in 4.3 of https://arxiv.org/abs/2003.03950 (and references therein)... the previous large max n_steps may have been derived from tests on the reference implementation, which may be less impacted by precision loss)